htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -1191,7 +1192,7 @@ public function doCollectOneCollector($mode = 0)
 				$client = $cm->make([
 					'host'           => $this->host,
 					'port'           => $this->port,
-					'encryption'     => 'ssl',
+					'encryption'     => !empty($this->imap_encryption) ? $this->imap_encryption : false,
```

Comment:
Property $imap_encryption must also be declared into the emailcollector.class.php to avoid undefined property error.